webpack.dev.ts

```diff
@@ -42,6 +43,11 @@ module.exports = merge(common, {
       },
     },
     client: {
+      progress: false,
```

Comment:
adds this back in https://github.com/influxdata/ui/pull/3411/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L21

Reply:
Yup, I totally misunderstood the errors that stemmed from a CLI command.